src/main/java/com/github/theholywaffle/teamspeak3/TS3Api.java

```diff
@@ -1002,9 +1048,6 @@ public void deleteChannelGroup(int groupId, boolean force) {
 	 * @see ChannelGroup#getId()
 	 * @see Permission#getName()
 	 */
-	public void deleteChannelGroupPermission(int groupId, String permName) {
```

Comment:
Would instead deprecate the method and describe what to use instead. Delegate the call in an adapted way to the new method.

Just deleting a method is a breaking change.

Reply:
I didn't pay attention, it's a mistake on my part to have deleted this method. I will correct it.